### PR TITLE
TCNForecaster example using V2 CLI

### PR DESCRIPTION
**Pull request by arun-rajora**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Add CLI v2 Example of forecasting Github Dau

fixes #

### CODE REVIEW

**Review by vlbejan — APPROVED** [review]

**Review by vlbejan — APPROVED** [review]

File: cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  type: mltable
+
+limits:
+  timeout_minutes: 60
+  max_concurrent_trials: 4
+  enable_early_termination: true
+
+forecasting:
+  time_column_name: "date"
+  forecast_horizon: 12
```
[the remark this answers is not in the document]
Comment (arun-rajora) [reply cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml]: Thanks for the feedback. I have updated the forecast horizon.

File: cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+limits:
+  timeout_minutes: 60
+  max_concurrent_trials: 4
+  enable_early_termination: true
+
+forecasting:
+  time_column_name: "date"
+  forecast_horizon: 14
+  frequency: "D"
```
Comment (nick863) [inline cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml:30]: We can remove frequency here for parity with the notebook.

Comment (arun-rajora) [reply cli/jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml]: Thanks! I have removed frequency from the config file.

File: cli/README.md [hunk 3]
```diff
@@ -165,6 +166,7 @@ path|status|description
...
+[jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml](jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau.yml)|[![jobs/automl-standalone-jobs/cli-automl-forecasting-task-github-dau/cli-automl-forecasting-task-github-dau](https://github.com/Azure/azureml-examples/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-github-dau-cli-automl-forecasting-task-github-dau/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-automl-standalone-jobs-cli-automl-forecasting-task-github-dau-cli-automl-forecasting-task-github-dau.yml)|A Time-Series Forecasting job using Github DAU dataset that trains only the TCNForecaster model.
```
Comment (nick863) [inline cli/README.md:169]: It seems, we have the same entry two times.

Comment (arun-rajora) [reply cli/README.md:168]: Thanks! I have updated it. It seems the readme.py generated duplicate values. Some duplicate values are also present in the main branch- [here](https://github.com/Azure/azureml-examples/blob/main/cli/README.md). Example- "jobs/automl-standalone-jobs/cli-automl-forecasting-bike-share/cli-automl-forecasting-task-bike-share.yml"

**Review by nick863 — APPROVED** [review]

**Pull request merged**